simtools/layout/layout_array_builder.py

```diff
@@ -0,0 +1,244 @@
+import astropy.units as u
+import matplotlib.patches as mpatches
+import numpy as np
+from astropy.table import QTable
+from matplotlib import pyplot as plt
+from matplotlib.collections import PatchCollection
+
+from simtools import io_handler
+from simtools.util import names
+from simtools.visualization import legend_handlers as leg_h
+
+
+class LayoutArrayBuilder:
+    """
+    Manage the array layout. It provides methods to create and plot a LayoutArray.
+    """
+
+    def __init__(self):
+        """Initialize LayoutArrayBuilder."""
+        self.io_handler = io_handler.IOHandler()
+        self.telescope_types = ["LST", "MST", "SCT", "SST"]
+        telescope_object_list = ["LSTObject", "MSTObject", "SCTObject", "SSTObject"]
+        telescope_colors = ["darkorange", "dodgerblue", "lightsteelblue", "black"]
+        self.colors_dict = {}
+        self.telescope_object_dict = {}
+        for step, telescope_type in enumerate(self.telescope_types):
+            self.colors_dict[telescope_type] = telescope_colors[step]
+            self.telescope_object_dict[telescope_type] = telescope_object_list[step]
+
+    @u.quantity_input(rotatio_angle=u.deg)
+    def plot_array(self, telescopes, rotate_angle=0 * u.deg):
```

Comment:
Do you really need to multiply by the unit here despite the decorator? (asking, not sure)

Reply:
yes, otherwise you pass an invalid argument as a default parameter